engine/src/main/java/org/terasology/rendering/ShaderManagerLwjgl.java

```diff
@@ -151,14 +145,13 @@ public void recompileAllShaders() {
         activeMaterial = null;
     }
 
-    @SuppressWarnings("unused")  // this is actually used, I'm not sure why the Jenkins Static Analyses says it's unused.
-    private GLSLMaterial prepareAndStoreShaderProgramInstance(String title, ShaderParameters params) {
+    // TODO: discuss having a `public unloadShaderProgram`, to dispose shader programs no longer in use by any node
+    public GLSLMaterial loadShaderProgram(String title) {
```

Comment:
Oh, I did not realize that.

Since you were nice enough to give me a reason, I will change this too :D